gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -41,6 +41,9 @@
     private static final String ECLIPSE_FORMATTING = "com.palantir.baseline-format.eclipse";
     private static final String PJF_PROPERTY = "com.palantir.baseline-format.palantir-java-format";
     private static final String GENERATED_MARKER = File.separator + "generated";
+    /** The implementation version of the palantir-java-format that we are recommending. */
+    private static final String PJF_IMPLEMENTATION_VERSION = "0.3.8";
```

Comment:
Can we either generate this value based on versions.lock, or add a test that it matches the locked version?

Reply:
This coordinate will not appear in `versions.lock`, neither in baseline nor in baseline users' projects.
It's not used at compile time, rather added as a dependency into a non-locked configuration.